static/js/settings.js

```diff
@@ -75,6 +75,9 @@ function setup_settings_label() {
         send_private_typing_notifications: $t({
             defaultMessage: "Display whether I'm typing private messages to recipients",
         }),
+        send_read_receipts: $t({
+            defaultMessage: "Display whether I've read messages to other users",
```

Comment:
This string is causing the failure. I think [this regex](https://github.com/zulip/zulip/blob/master/tools/lib/capitalization.py#L181) that checks for capitalization errors is buggy but I'd expect the above `I'm` strings to fail too.

Any ideas here?

Reply:
There’s a specific [exclusion](https://github.com/zulip/zulip/blob/9fec1fb16d98da9e9dbb44a19ea87b562bc82935/tools/lib/capitalization.py#L77) for “I'm”; add one for “I've”.

Reply:
Thanks! That fixed it.